scripts/ticketmaster-reverse-geocode.py

```diff
@@ -0,0 +1,63 @@
+import requests
+import mysql.connector as mysql
+import configparser
+import googlemaps
+import time
+
+### read-in data need for sql connection
+config = configparser.ConfigParser()
+config.read('../config.ini')
+
+db = mysql.connect(
+    host=config['mysql']['host'],
+    user=config['mysql']['user'],
+    passwd=config['mysql']['pass'],
+    database="state_of_music"
+)
+cursor = db.cursor(buffered=True)
+
+cursor2 = db.cursor(buffered=True)
+
+gmaps = googlemaps.Client(key='AIzaSyBcYrsWYbXhwxJ-NQXUSZs6aCoCaiQ-97o')
```

Comment:
[RC]
Def need to move the key to `config.ini`, totally my bad sorry!!

Reply:
Moved!